glide.yaml

```diff
@@ -4,17 +4,21 @@ import:
 - package: k8s.io/api
   version: kubernetes-1.14.0
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  repo:    https://github.com/openshift/kubernetes-apimachinery.git
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/client-go
   version: kubernetes-1.14.0
 - package: k8s.io/component-base
   version: kubernetes-1.14.0
 - package: k8s.io/kubernetes
   repo:    https://github.com/openshift/kubernetes.git
   version: origin-4.2-kubernetes-1.14.0
+- package: k8s.io/cli-runtime
+  repo:    https://github.com/openshift/kubernetes-cli-runtime.git
+  version: origin-4.2-kubernetes-1.14.0
 # 2. openshift
 - package: github.com/openshift/api
-  version: master
+  version: release-4.2
```

Comment:
If you're switching api to `release-4.2`, switch `client-go` as well.

Reply:
Ah, didn't check for that.  Changing it.

Reply:
Ah, `openshift/client-go` isn't `k8s.io/client-go`.  Pinning it.